### PR TITLE
[python] Use larger default py.init_buffer_bytes

### PR DESCRIPTION
**Pull request by johnkerl**
This improves query performance

### CODE REVIEW

File: apis/python/src/tiledbsoma/tiledb_object.py [hunk 1]
```diff
@@ -55,6 +58,17 @@ def __init__(
...
         # Null ctx is OK if that's what they wanted (e.g. not doing any TileDB-Cloud ops).
 
+    def _default_ctx(self) -> tiledb.Ctx:
+        """
+        Default TileDB configuration parameters, for when none other has been specified by the
+        user.
+        """
+        return tiledb.Ctx(
+            {
+                "py.init_buffer_bytes": 1 * 1024**3,
```
Comment (gspowley) [inline apis/python/src/tiledbsoma/tiledb_object.py:68]: Can you please add a line for `soma.init_buffer_bytes` with the same value?

FYI, we may converge these into one parameter in the future.

**Review by gspowley — DISMISSED** [review]

File: apis/python/src/tiledbsoma/tiledb_object.py (hunk 1 above)
Comment (johnkerl) [reply apis/python/src/tiledbsoma/tiledb_object.py:68]: @gspowley I would happily but this is for `main-old`! :)

Comment (gspowley) [reply apis/python/src/tiledbsoma/tiledb_object.py:68]: Sorry, didn't notice that the first time. :)

**Review by gspowley — APPROVED** [review]

**Comment by johnkerl** [comment]
The C++ SOMA reader has its own config

**Pull request closed without merging**